clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/config/TokenAuth.java

```diff
@@ -31,9 +32,9 @@
 
   default Optional<String> getTokenAsString() {
     String token = null;
-    if (StringUtils.isNotEmpty(getTokenFile())) {
+    if (!getTokenFile().isEmpty()) {
```

Comment:
In this case I'd lean more towards having getTokenFile and getToken return Optionals than never-null Strings where empty has a special meaning - it feels like Optional captures the intent better. WDYT?

Reply:
That's a good point, if it really is optional we should just handle that in the constructor; I'll make that change.